packages/compiler/src/core/cli/actions/init.ts

```diff
@@ -13,6 +15,9 @@ export async function initAction(
 ): Promise<readonly Diagnostic[]> {
   try {
     await initTypeSpecProject(host, process.cwd(), args);
+    if (!args.skipInstall) {
```

Comment:
I think we need this to just be a prompt on the user to ask to confirm(specially as right now it runs npm install forcably)

Reply:
@allenjzhang do you have a preference?

Reply:
@mario-guerra is looping in UX designer and other feedback sources to investigate any further optimize the CLI UX experience. May change then. Let's do the prompt for now. Thx!

Reply:
my opinion is that we should not do any install without asking the user. A lot of people will run this with other package manager and this might just setup an invalid node_modules for them. It is quite common for similar tool to ask if you want to run install at the end.

Reply:
Added the confirmation, but now it seems that it may be redundant. With the confirmation, I think we could eliminate the flag... because right now if you supply `--install` it is still going to ask you to confirm... which seems a little silly.

Reply:
I could also change it so it only confirms if you don't specify either flag (`--install` or `--no-install`)

Reply:
yeah Ithinkj if the flag(on or off) is set then we bypass the confirmation